config-templates/ibm/ocs/4.6_local/custom-parameters.json

```diff
@@ -0,0 +1,63 @@
+[
+  {
+    "description": "Cluster Name of OCS",
+    "displayname": "Cluster Name",
+    "name": "ocs-cluster-name",
+    "required": "true"
+  },
+  {
+    "description": "Enter the OSD device path. Mandatory",
+    "displayname": "Osd Device Path",
+    "name": "osd-device-path",
+    "required": "true",
+    "type": "csv"
+  },
+  {
+    "description": "Enter the MON device path. Mandatory",
+    "displayname": "Mon device Path",
+    "name": "mon-device-path",
+    "required": "true",
+    "type": "csv"
+  },
+  {
+    "description": "Enter the number of OSD. Optional. Default is 1",
+    "displayname": "Num of osd",
+    "name": "num-of-osd",
+    "default": "1",
+    "required": "false"
+  },
+  {
+    "description": "Worker Nodes you want to deploy OCS on. Min 3. Leave if you want OCS on all workers",
+    "displayname": "worker nodes",
+    "name": "worker-nodes",
+    "required": "false",
```

Comment:
As this parameter is optional, have you tested  without giving any value from cli for this ?